Info: resolving _kpathsea_bug_address by linking to __imp__kpathsea_bug_address (auto-import/home/janneke/vc/gub/target/cygwin/root/usr/cross/bin/i686-cygwin-ld: warning: auto-importing has been activated without --enable-auto-import specified on the command line.
This should work unless it involves constant data structures referencing symbols from auto-imported DLLs.
kpsewhich.o: In function `read_command_line':
/home/janneke/vc/gub/target/cygwin/src/texlive-2009/texk/kpathsea/kpsewhich.c:472: undefined reference to `_str_list_add'
collect2: ld returned 1 exit status
)

--- texlive-2009/texk/kpathsea/str-list.h~	2009-03-16 16:13:07.000000000 +0100
+++ texlive-2009/texk/kpathsea/str-list.h	2010-03-24 14:57:44.304239167 +0100
@@ -37,25 +37,25 @@ typedef struct
 #define STR_LIST_LAST_ELT(l) STR_LIST_ELT (l, STR_LIST_LENGTH (l) - 1)
 
 /* Return a new, empty, list.  */
-extern str_list_type str_list_init (void);
+extern KPSEDLL str_list_type str_list_init (void);
 
 /* Append the string S to the list L.  It's up to the caller to not
    deallocate S; we don't copy it.  Also up to the caller to terminate
    the list with a null entry.  */
-extern void str_list_add (str_list_type *l, string s);
+extern KPSEDLL void str_list_add (str_list_type *l, string s);
 
 /* Append all the elements from MORE to TARGET.  */
-extern void str_list_concat (str_list_type * target, str_list_type more);
+extern KPSEDLL void str_list_concat (str_list_type * target, str_list_type more);
 
 /* Free the space for the list elements (but not the list elements
    themselves).  */
-extern void str_list_free (str_list_type *l);
+extern KPSEDLL void str_list_free (str_list_type *l);
 
 /* Append each element of MORE to each element of TARGET.  */
-extern void str_list_concat_elements
+extern KPSEDLL void str_list_concat_elements
   (str_list_type *target, str_list_type more);
 
 /* Remove duplicate elements from L, freeing their space.  */
-extern void str_list_uniqify (str_list_type *l);
+extern KPSEDLL void str_list_uniqify (str_list_type *l);
 
 #endif /* not KPATHSEA_STR_LIST_H */
